docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -511,6 +511,8 @@ If a name is not specified, the name section is not included in the client ID. F
 The key-value storage to use. Use this option only if using a storage system provided with a
 plugin. The default is `rocksdb`.
 
+For ease of use when using a debugger, use the `memory` option.  
```

Comment:
Add a link to the option?

Reply:
https://github.com/hyperledger/besu/pull/1617 ?

I think this needs more caveats that tell the casual reader "You probably don't want this option".  The release notes say "Added `memory` as an option to `--key-value-storage`.  This ephemeral storage is intended for sync testing and debugging. "

Reply:
Updated to make this clearer

Reply:
@NicolasMassart - I don't understand your comment?